src/stepfunctions/steps/compute.py

```diff
@@ -12,10 +12,32 @@
 # permissions and limitations under the License.
 from __future__ import absolute_import
 
+from enum import Enum
 from stepfunctions.steps.states import Task
 from stepfunctions.steps.fields import Field
-from stepfunctions.steps.utils import resource_integration_arn_builder
-from stepfunctions.steps.integration_resources import IntegrationPattern, IntegrationServices, LambdaApi, GlueApi, BatchApi, EcsApi
+from stepfunctions.steps.utils import get_service_integration_arn
+from stepfunctions.steps.integration_resources import IntegrationPattern
+
+Lambda = "lambda"
+Glue = "glue"
+Ecs = "ecs"
+Batch = "batch"
```

Comment:
```suggestion
LAMBDA_SERVICE_NAME = "lambda"
GLUE_SERVICE_NAME = "glue"
ECS_SERVICE_NAME = "ecs"
BATCH_SERVICE_NAME = "batch"
```

nit: Variable names should written be in `snake_case`. Class names are `PascalCase`. Constants in `CAPITAL_SNAKE_CASE`. Repeats.